src/ListView/ListViewItem.js

```diff
@@ -0,0 +1,113 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import {
+  ListViewExpand,
+  ListGroupItem,
+  ListGroupItemHeader,
+  ListGroupItemContainer
+} from './index'
+import ListViewRow from './ListViewRow'
+
+/**
+ * ListViewItem - main ListViewItem component which handles the expansion logic.
+ * ListViewItem is considered expandable if it has child props. In that case it
+ * renders ListGroupItemHeader and ListGroupItemContainer
+ */
+export default class ListViewItem extends React.Component {
+  constructor() {
+    super()
+    this.state = { expanded: false }
```

Comment:
question - do we prefer state object defined inside the constructor or outside within the class? We don't have a linter checking this right now, but it may be good to introduce it at some point for consistency... maybe we can consider more lint settings in a future PR?

Reply:
All examples in ReactJS doc initialize the state in constructor.

Reply:
@jtomasek I'm good with this pattern (just wanted to note it though). Will update my other PRs accordingly and follow this standard.  Thanks!